File: app/src/main/java/com/buhzzi/wuej_0/kit/font_constants.java
package com.buhzzi.wuej_0.kit;

import android.content.Context;
import android.graphics.Typeface;

import androidx.core.content.res.ResourcesCompat;

import com.buhzzi.wuej_0.R;

public class font_constants {
	public static Typeface xeu_tf;
	public static void init(Context ctx) {
		xeu_tf = ResourcesCompat.getFont(ctx, R.font.xeufont_ascii);
	}
}

File: app/src/main/java/com/buhzzi/wuej_0/kit/FontConstants.kt
package com.buhzzi.wuej_0.kit

import android.graphics.Typeface
import androidx.core.content.res.ResourcesCompat
import com.buhzzi.wuej_0.R


class FontConstants {
	companion object {
		val xeuTf: Typeface
			get() = ResourcesCompat.getFont(StackedActivity.topActivity, R.font.xeufont_ascii)!!
	}
}
